mods/farming/nodes.lua

```diff
@@ -74,9 +80,19 @@ minetest.register_node("farming:soil", {
 
 minetest.register_node("farming:soil_wet", {
 	description = S("Wet Soil"),
-	tiles = {"default_dirt.png^farming_soil_wet.png", "default_dirt.png^farming_soil_wet_side.png"},
+	tiles = {
+		"default_dirt.png^farming_soil_wet.png",
+		"default_dirt.png^farming_soil_wet_side.png"
+	},
 	drop = "default:dirt",
-	groups = {crumbly=3, not_in_creative_inventory=1, soil=3, wet = 1, grassland = 1, field = 1},
+	groups = {
+		crumbly = 3,
+		not_in_creative_inventory = 1,
+		soil = 3,
+		wet = 1,
+		grassland = 1,
+		field = 1
+	},
```

Comment:
No other mods use similar group formatting. Are you sure this is better?

Reply:
No, I'm not. Since group names are small, it looks better in two lines.